app/scripts/views/mixins/password-mixin.js

```diff
@@ -45,6 +46,24 @@ define(function (require, exports, module) {
         // IE8 blows up when changing the type of the input field. Other
         // browsers might too. Ignore the error.
       }
+    },
+
+    onPasswordKeypress: function (event) {
+      var val = this.getElementValue('.password').length;
+      if (val < 8) {
+        this.setPasswordWarning(true);
+      } else {
+        this.setPasswordWarning(false);
+      }
+    },
+
+    setPasswordWarning: function (isVisible) {
```

Comment:
Can you split this into two functions, one "show" and one "hide" variant? Maybe "showPasswordHelper" and "hidePasswordHelper"? Passing in booleans for flow control "works" but means the caller has to think "what do I have to pass in to show?" or "what do I have to pass in to hide?" Forcing that choice to be made when choosing the function is simpler.